CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.1.0 November 3, 2021
+
+## Features ✨ and improvements 🏁
+* Set thread priorities and set thread CPU affinity based on the thread's priority ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
+* Avoid spawning extra AssetManagerFileSource threads when multiple instances of a MapView is created during application lifecycle ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
+
+## Bug fixes 🐞
+* Fix rendering artifact when some of the model layer models may have wrong placement when globe view projection is used ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
+* Fix rare heatmap flickering when zooming the map ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
+* Fix an issue where an Observable event could be dispatched on a thread, different from the subscription thread ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
+* Fix an issue where promoteId parameter for VectorSource was overwritten when source tilejson is loaded ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
```

Comment:
fixes related to `promoteId ` and `glyph drawing`, `CPU affinity` were already mentioned in 10.1.0-beta.1 and rc.1, are these fixed again by gl-native or core?

Reply:
@ank27 no they are not fixed again but taking into account that we're GA now we share all the updates (from beta, rc and stable) in this changelog. That will make easier for customers to see what changed between 10.0.0 and 10.1.0

@Chaoba taking it into account - can you append entries related to our SDK only from beta and rc release as well in this changelog?

Reply:
No, just checked they are the same pr, but was included in the chagnelog of gl-native.  Will remove this line.

Reply:
@kiryldz thanks for clearing that up 😄 .

Reply:
```suggestion
* Fix an issue where promoteId parameter for VectorSource was overwritten when source tilejson is loaded. ([#810](https://github.com/mapbox/mapbox-maps-android/pull/810))
```